From ec9865c5b2a05908dff774eda352d84d6a001e1a Mon Sep 17 00:00:00 2001
From: Lee Duncan <lduncan@suse.com>
Date: Sat, 2 Dec 2017 14:24:52 -0800
Subject: [PATCH] Declare inline best_match_bufcmp() as static.

The 7.2.1 version of gcc seems to be more strict
about this.  See https://gcc.gnu.org/onlinedocs/gcc/Inline.html
---
 iscsiuio/src/uip/ipv6.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/iscsiuio/src/uip/ipv6.c b/iscsiuio/src/uip/ipv6.c
index ced98a67..05efa73f 100644
--- a/iscsiuio/src/uip/ipv6.c
+++ b/iscsiuio/src/uip/ipv6.c
@@ -49,7 +49,7 @@
 #include "dhcpv6.h"
 #include "ping.h"
 
-inline int best_match_bufcmp(u8_t *a, u8_t *b, int len)
+static inline int best_match_bufcmp(u8_t *a, u8_t *b, int len)
 {
 	int i;
 
